### PR TITLE
Update docs to indicate LOAD DATA...REPLACE INTO support

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Update docs to indicate LOAD DATA...REPLACE INTO support

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- https://github.com/pingcap/docs/pull/7426
- https://github.com/pingcap/tidb/pull/31303

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by morgo — APPROVED** [review]

**Comment by TomShawn** [comment]
From which version is this supported? v6.0?

**Review by qw4990 — APPROVED** [review]

File: privilege-management.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ Requires the `DROP` privilege for the table.
 
 ### LOAD DATA
 
-Requires the `INSERT` privilege for the table.
+Requires the `INSERT` privilege for the table. When using `REPLACE INTO` the `DELETE` privilege is also required.
```
Comment (TomShawn) [inline privilege-management.md:296]:
```suggestion
Requires the `INSERT` privilege for the table. When you use `REPLACE INTO`, the `DELETE` privilege is also required.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**